### DISCUSSION TITLE
What's the minimum reserved space

### DISCUSSION THREAD

**Discussion by Josh65-2201** [ventoy/Ventoy#1026] [Q&A] [answered]
I'm trying to make a 15GB boot file area on my storage drive that's 2TB. When entering 2032 GB in reserved space it says it's invalid it also does it with 2020GB

**Comment by steve6375** [comment, marked as answer]
Most 2TB drives are less than 2000GiB in size
Note that manufacturers use '1TB' =  1,000,000,000,000 bytes
whereas 1TiB = 1024 x 1024 x 1024 x 1024 =  1 099 511 627 776 bytes
see https://www.quora.com/What-is-the-difference-between-Terabyte-TB-and-Tebibyte-TiB
Just try a smaller size for the reserved space and see what you end up with.

**Reply by Josh65-2201** [reply]
~~It doesn't seem to be a conversion issue since I can put in 1500GB and it preservers it at that, same with any other accepted value.~~

Edit: Never mind i was not reading maximum windows stated. I instead used ventoy reported max size of the drive.
